internal/pkg/daemon/enricher/container.go

```diff
@@ -124,6 +120,10 @@ func (e *Enricher) getContainerInfo(
 						recordProfile: recordProfile,
 					}
 
+					if e.labelProblematicContainers {
+						go labelContainer(e.impl, e.clientset, *info, e.logger)
```

Comment:
We can make this a method to avoid passing `*e`'s variables. :)

Reply:
true! I'll do that